main.cpp

```diff
@@ -1,7 +1,216 @@
+
+ Wait for my code review.
+ */
+
+//2)
+void variableDeclarations()
+{
+    //example:
+    int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
+    int age = 16;
+    int time = 100;
+
+    char firstLetter = 'a'; 
+    char secondLetter = 'b';
+    char thirdLetter = 'c';
+
+    bool lie = false;
+    bool disabled = false;
+    bool skilled = false;
+
+    float money = 2.55f;
+    float numberPi = 3.1415f;
+    float altitude = 1555.302f;
+
+    double extraPrecisePi = 3.1415192654;
+    double someAmount = 4.00000001;
+    double everestAltitude = 8.848;
+    
+    
+    
+    ignoreUnused(number, age, time, firstLetter, secondLetter, thirdLetter, lie, disabled, skilled, money, numberPi, altitude, extraPrecisePi, someAmount, everestAltitude ); //passing each variable declared to the ignoreUnused() function
+}
+/*
+ 10 functions
+ example:
+ */
+bool rentACar(int rentalDuration, int carType = 0)  //function declaration with random number of arguments, arbitrary number of arguments have default value
+{ 
+    ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused() function
+    return {}; //if your function returns something other than void, add 'return {};' at the end of it.
+} 
+
+/*
+ 1)
+ */
+void setSalary(int wage, int deduction, bool married)
+{
+    ignoreUnused(wage, deduction, married);
+}
+
+/*
+ 2)
+ */
+int getSalary(int salary = 2000)
+{
+    ignoreUnused(salary);
+    return{};
+}
+
+/*
+ 3)
+ */
+int howManyMiles(int kmph, double gas, double time)
+{
+    ignoreUnused(kmph, gas, time);
+    return{};
+}
+
+/*
+ 4)
+ */
+
+bool isLying(int pulse, int galvanicSkinResponseValue = 2, int pupilDilation = 4)  
+{
+    ignoreUnused(pulse, galvanicSkinResponseValue, pupilDilation);
+    return{};
+}
+
+/*
+ 5)
+ */
+char guessTheLetter(char userInputValue)
+{
+    ignoreUnused(userInputValue);
+    return{};
+}
+
+/*
+ 6)
+ */
+char randomizeAcronym(char firstLetter, char secondLetter, char thirdLetter = 'z')
+{
+    ignoreUnused(firstLetter, secondLetter, thirdLetter);
+    return {};
+}
+/*
+ 7)
+ */
+bool isGoodBoi(bool dog = true, bool playful = true)
+{
+    ignoreUnused(dog, playful);
+    return{};
+}
+/*
+ 8)
+ */
+void songCatalog(int someSong = 5)
+{
+    ignoreUnused(someSong);
+}
+/*
+ 9)
+ */
+void printCurrentGasInTank(int tankNumber)
+{
+    ignoreUnused(tankNumber);
+}
+/*
+ 10)
+ */
+bool isAfk(int numberOfClicks, int numberOfKeyStrokes)
+{
+    ignoreUnused(numberOfClicks, numberOfKeyStrokes);
+    return{};
+}
+
 int main()
 {
+    //example of calling that function
+    rentACar(6, 2); 
+    //1)
+    setSalary(2000, 100, true);
+    //2)
+    getSalary();
+    //3)
+    howManyMiles(200, 50.552, 80.5);
+    //4)
+    isLying(120);
+    //5)
+    guessTheLetter('a');
+    //6)
+    randomizeAcronym('B', 'H', 'P');
+    //7)
+    isGoodBoi(true);
+    //8)
+    songCatalog();
+    //9)
+    //10)
+    printCurrentGasInTank(2);
```

Comment:
Can you make these functions that return values, store the returned value in a variable, and pass those variables to the `ignoreUnused` function at the end?
example:
```
bool isAwayFromKeyboard = isAfk(1, 2);

ignoreUnused(isAwayFromKeyboard);
```